### ISSUE TITLE
Multiline blockquotes support

### ISSUE THREAD

**Issue by olayway** [flowershow/flowershow#42]
In Obsidian, you can create multiline blockquotes by returning at the end of the first line and starting the next one with `>` (which is added automatically in Obsidian by hitting enter while being on the end of the first line.)

**Raw example**:
```md
> Roses are red,
> violets are blue.
```

**How it renders in Obsidian:**
![image](https://user-images.githubusercontent.com/52197250/179528836-7597fcba-0c3a-42bd-8fff-7a602f29a2ee.png)

**How it renders here on GitHub**:
> Roses are red,
> violets are blue.

**How it renders in Flowershow:**
![image](https://user-images.githubusercontent.com/52197250/179529287-1be12213-6612-4c16-bc6f-6fcf1f416b45.png)

## What works
Adding two spaces at the end of the first line.

**Raw example**:
```md
> Roses are red,  
> violets are blue.
```

**How it renders in Obsidian:**
![image](https://user-images.githubusercontent.com/52197250/179528836-7597fcba-0c3a-42bd-8fff-7a602f29a2ee.png)

**How it renders here on GitHub**:
> Roses are red,  
> violets are blue.

**How it renders in Flowershow:**
![image](https://user-images.githubusercontent.com/52197250/179531262-67a5bdca-23d4-4fe2-817f-0bb2243ca2c2.png)

**Comment by rufuspollock** [comment]
That is correct (commonmark) markdown if no trailing space on first line. For github flavored markdown it should though not sure about how the gfm support in remark works. 

> Rose are red
> Violets are blue

A proper test would be the following which i think works.

```
> Rose are red
>
> Violets are blue
```

> Rose are red
>
> Violets are blue

Aside: good to add an example of the docs page for the relevant feature and see what happens.

**Comment by olayway** [comment]
> A proper test would be the following which i think works.
> ```md
> > Rose are red
> >
> > Violets are blue
> ```
>
> Rose are red
>
> Violets are blue

But the above lines end up separated by an additional empty line...

Since we are mostly targetting Obsidian users, I think this is worth taking into account how it works in Obsidian. (+ it seems to work the same here on GitHub, although I couldn't find confirmation for this in the GFM spec).

> Aside: good to add an example of the docs page for the relevant feature and see what happens.

I've noticed it **while** working on such a page (demo for markdown/obsidian syntax support), but I'll add a link here after merge. But I've just updated the description of this issue, so should be more clear what happens.

**Comment by Gutts-n** [comment]
@olayway in tests here, it looks like it's working correctly, am I right? 


![image](https://github.com/datopian/flowershow/assets/57202549/78a3c6e3-db36-4992-a910-1c2eb90e84e0)

**Comment by rufuspollock** [comment]
FIXED. it seems ...
